governance-exercises/MagicRWA Standing order changes.md

```diff
@@ -0,0 +1,60 @@
+---
+title: MagicRWA Standing Order Changes - July 24, 2023
+summary: Signal your support or opposition to the proposed change to the MagicRWA Standing Order.
+discussion_link: https://forum.makerdao.com/t/xxx
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-31T16:00:00
+end_date: 2023-08-02T16:00:00
+---
+# Poll: MagicRWA Standing Order Changes - July 24, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 31 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following change:
+* Approve a change to the standing order that pertains to the transfer of RWA revenues from vaults arranged by MagicRWA, which will be carried out by the trustees.
+
+In summary, the proposed change will direct the trustees of the MagicRWA arrangements to return revenue quarterly through the conduit and jar contract.
+
+The standing order can be viewed [here](link to standing order text on Pinata)
+
+Please review the full discussion [thread](https://forum.makerdao.com/txxx) to help inform your position before voting.
+
+**Note**: The existing standing order for the MagicRWA vault trust related to ALM transfers to maintain the DAI peg is invalidated as a result of this proposed standing order. The ALM order will be replaced shortly once the new version has been fully drafted. A forum [poll](link to forum poll) has been created to help judge community sentiment on the logic for the replacement ALM order.
+
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This proposed standing order change will be included in an upcoming Executive Vote.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
+
+---
+
+## Resources
+
+Additional Information about MagicRWA vaults can be found [here](link)
```

Comment:
Yep, optimistic to assume there will be a nice URL you can link to in my experience. But good to try to do this if you can.